scripts/update-drafts.ts

```diff
@@ -137,12 +138,29 @@ async function main() {
       continue;
     }
 
-    const spec_url = feature.data.__compat.spec_url;
-    if (!spec_url) {
-      continue;
+    let parent_url = null;
+    if (!feature.data.__compat.spec_url) {
```

Comment:
The `spec_url` getter will always give you an array, which is a little tidier:

```suggestion
    if (!feature.spec_url.length) {
```